src/components/tiles/TilesList.tsx

```diff
@@ -43,6 +44,7 @@ export const TilesList = ({
   return (
     <ScrollView testID={tID('tiles-list')} style={styles.scrollView}>
       <View style={styles.view}>
+        <TrackTile />
```

Comment:
Thinking we should pass the TrackTile into the TileList via the children prop so this list remains fully configurable.

Reply:
Done.